### ISSUE TITLE
interaction note resolving strangely

### ISSUE THREAD

**Issue by ValWood** [pombase/website#1719]
https://www.pombase.org/gene/SPAC25A8.01c

Don't have time right now to track the discussion, but I thought we were only going to display the notes with a mouse-over?

**Comment by kimrutherford** [comment]
Here's the discussion: https://github.com/pombase/website/issues/40#issuecomment-616991791

I can't remember now but probably I implemented it as a "view..." button/link because I wasn't which part of the annotation you should mouse-over to get the note.

**Comment by ValWood** [comment]
OK. The genetic interaction notes seem to. be OK on the page above, but the physical interaction notes overflow the table.

**Comment by kimrutherford** [comment]
Could you send a screenshot?  I can't see any overflowing bits in my browser.

**Comment by ValWood** [comment]
Actually I think this was a known issue from way-back....
It isn't the note field, it is the page width generally. It happens to any columns as I narrow my browser...


<img width="545" alt="Screenshot 2021-07-05 at 22 41 39" src="https://user-images.githubusercontent.com/7359272/124519813-67dfce00-dde2-11eb-824f-7dfe4343ccef.png">


<img width="432" alt="Screenshot 2021-07-05 at 22 41 59" src="https://user-images.githubusercontent.com/7359272/124519816-6a422800-dde2-11eb-9d63-10866b6ab039.png">

**Comment by ValWood** [comment]
It's not a big issue....

**Comment by kimrutherford** [comment]
It's tricky to get fine control over the formatting of HTML tables.  I think the problem in this case is the product "cyclohydrolase/phosphoribosylaminoimidazolecarboxamideformyltransferase"

The browser won't let the product column get narrower than that string.  I've changed the HTML style for that column so that it will break the word over to the next line if it has to.  I've also tweaked a few other things so that it will work better when the browser window is narrow.

The changes should be visible in the morning.
